components/base/Collapsible.tsx

```diff
@@ -78,15 +85,33 @@ function Collapsible({ triggerText, children }: Props) {
 
   const transitionTime = 200;
 
+  // Sets active state on collapsible
+  function handleActive() {
+    if (!active) {
+      setActive(true);
+    } else {
+      setActive(false);
+    }
```

Comment:
This could just be `setActive(!active)`

Reply:
Yo! Good point! I was really struggling with the eradicating the redundancy here with these nested if statements. Thanks for the free point ;)